doc/doxygen/riot.doxyfile

```diff
@@ -827,7 +827,11 @@ EXCLUDE_PATTERNS       = */board/*/tools/* \
                          */cpu/native/osx-libc-extra \
                          */cpu/x86/include/* \
                          */drivers/kw2xrf/include/overwrites.h \
-                         */pkg/*/*/* \
+                         */pkg/emb6/* \
+                         */pkg/lwip/* \
+                         */pkg/nordic_softdevice_ble/* \
+                         */pkg/openthread/* \
+                         */pkg/semtech-loramac/* \
```

Comment:
Why not `*/pkg/*/*` so every `pkg` is included?

Reply:
Well, it was already like that. The purpose is to not _exclude_ all packages directory (the EXCLUDE_PATTERNS list is updated here)

Reply:
Ah sorry ^^"